hooks/post_user_create

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 app="wallabag2"
+user="wallabag2"
```

Comment:
Of course, the user was not good, but neither here in the case of multi-instance.

The user should be set during the install (or upgrade), to the value of `$app`. If you don't understand or you don't know how to do it, I can help you.